appgate/syncer/git.py

```diff
@@ -201,18 +203,27 @@ def create_or_update_pull_request(
         raise NotImplementedError()
 
 
-def clone_repo(ctx: GitOperatorContext, vendor: str, repository_path: Path) -> Repo:
+def clone_repo(ctx: GitOperatorContext, repository_path: Path) -> Repo:
     repository = ctx.git_repository_fork or ctx.git_repository
     log.info("[git-operator] Initializing the git repository by cloning %s", repository)
     if repository_path.exists():
         shutil.rmtree(repository_path)
     if not GIT_SSH_KEY_PATH.exists():
         raise AppgateException(f"Unable to find SSH key {GIT_SSH_KEY_PATH}")
+    url = (
+        f"git@{urlparse(ctx.git_hostname).hostname}:{repository}"
```

Comment:
In theory its possible to specify a non-standard git+ssh port for gitlab, we should support that.
See https://about.gitlab.com/blog/2016/02/18/gitlab-dot-com-now-supports-an-alternate-git-plus-ssh-port/

That could be done in a follow up pr if you want though.

Reply:
I agree with @thomascellerier - I ran into this immediately since I have GitLab running in Docker on a Linux host, so the GitLab ssh port needs to be something other than 22.  For testing, I can target the Docker IP of GitLab since GitLab and minikube (sdp-operator) are on the same host.

Reply:
Added support for alternate ssh port